engine/core/IgeCellSheet.ts

```diff
@@ -15,7 +14,7 @@ export class IgeCellSheet extends IgeTexture {
 	_cellRows: number = 0;
 	_cellWidth: number = 0;
 	_cellHeight: number = 0;
-	_sheetImage?: IgeImage | IgeCanvas;
+	_sheetImage?: IgeImage
```

Comment:
Are we sure we don't want to support IgeCanvas as a sheet image here? I can't remember requirements but it was in the original engine code so I'm just checking :)

Reply:
IgeImage type is now defined as

`export type IgeImage = ImageBitmap | IgeCanvas;`

so we should still have this covered